upgrading-open-data-hub.adoc

```diff
@@ -0,0 +1,33 @@
+:_module-type: ASSEMBLY
+
+ifdef::context[:parent-context: {context}]
+
+:context: upgrade
+
+[id="upgrading-open-data-hub_{context}"]
+= Upgrading Open Data Hub
+
+[role='_abstract']
+You can upgrade the Open Data Hub Operator from version 1 to version 2 from the OpenShift web console. 
+For information about installing the Open Hub Operator version 2, see link:{odhdocshome}/installing-open-data-hub/#installing-odh-v2_installv2[Installing Open Data Hub version 2].
+
+Upgrading Open Data Hub involves the following tasks:
+
+. Upgrade the Open Data Hub Operator.
+. Install Open Data Hub components.
+. Access the Open Data Hub dashboard.
```

Comment:
Because the intro to this list is a gerund phrase ("Upgrading Open Data Hub involves..."), I think the bullet list should also use that style:

- Upgrading...
- Installing...
- Accessing...